_data/glossary/service.yaml

```diff
@@ -0,0 +1,9 @@
+id: service
+name: Service
+tags:
+- fundamental
+- core-object
+short-description: |
+  An API object that manages load-balanced access to a replicated set of Pods.
```

Comment:
An API object that describes how to access applications, such as a set of Pods, and can describe ports and load-balancers.

Reply:
thx